<?php
	include("beginning.php");
?>
	<div class="grid_1 alpha" id="left-part">
					<img class="stamps-logo" src="/images/stamps_logo.gif" alt="STAMPS logo">
	</div>
	<div class="grid_8 omega" id="right-part">
					<h2>August 3 – August 13, 2011</h2>
				
					<p>Course directors <a target="_blank" href="http://jbpc.mbl.edu/labs-msogin.html">Mitchell Sogin</a> and <a target="_blank" href=" http://jbpc.mbl.edu/labs-dmarkwelch.html">David Mark Welch</a></p>
					
					<p><a href="stamps_calendar.php"> Schedule</a>
						<!-- </br>  -->
						 <!-- Information for students </br> 
						 Information for faculty -->
					</p>	
	</div>
				<div class="clear">&nbsp;</div>
				<div class="grid_10" id="home-text">
				<p>Deep DNA sequencing using massively-parallel, next-generation technology has enabled nearly comprehensive environmental surveys that can describe the different kinds of microbes in a community and their relative abundance. These descriptions of richness and evenness make possible estimates of microbial diversity, but the size of the required data sets pose enormous computational challenges. The rapidly expanding flow of information from next generation DNA sequencing platforms has fueled healthy debate about best practices for data analysis while at the same time building a user demand for tools that can address important ecological questions. The STAMPS course will promote dialogue and the exchange of ideas between experts in analysis of metagenomic data and offer interdisciplinary bioinformatic and statistical training to practitioners of molecular microbial ecology and genomics.
				</p><p>
				Topics to be covered include but are not limited to next-generation strategies for analysis of microbial communities; acquisition and organization of next generation sequence data; principles of quality control of sequence data; the theory of cluster and rarefaction analyses; taxonomic assignments for high-throughput data; statistical models for estimating microbial diversity; microbial community comparison methodology and metrics; and analysis of shotgun metagenomic data. The lectures will be augmented by a computer laboratory where students will have the opportunity to use state of the art facilities to test theoretical concepts described in the lectures series. Guided by authors of ARB, CatchAll, Esprit, Mothur, Pyronoise, RDP, Silva, UClust, and Pyronoise for cluster analyses; mother, QIIME, Unifrac, and VAMPS for community analysis; and METAREP and MG-RAST for shotgun metagenomics analysis, participants in the workshop will have the opportunity to compare different analysis techniques for molecular data.
				</p><p>
				The course is designed for established investigators, postdoctoral fellows and advanced graduate students from diverse biological fields. The Workshop Participants will also be given priority for acceptance into the MBL’s Workshop in Molecular Evolution that will meet the preceding week. This option will provide an opportunity to develop a stronger conceptual framework for understanding the basis of analytical tools in the STAMPS workshop.
				</p>
		<!-- </div> -->
<?php
	include("end.php");
?>